### PR TITLE
Enzyme and jest test

### PR DESCRIPTION
**Pull request by ryanminjaeshin**
* Run basic test with Jest
* Update webpack

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react"
+  ],
+  "plugins": [
+    [
+      "@babel/plugin-proposal-class-properties"
```
Comment (rickboot) [inline .babelrc:8]: What does this do?

File: .eslintignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 bundle.js
-main.bundle.js
\ No newline at end of file
+main.bundle.js
+App.spec.js
```
Comment (rickboot) [inline .eslintignore:3]: I see you remembered Eliza's bundle.js warning. Do we need to exclude tests?

File: .vscode/settings.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
   "cSpell.words": [
+    "preve",
```
Comment (rickboot) [inline .vscode/settings.json:3]: Is vscode or a plugin nagging about spelling?

File: client/src/App.jsx [hunk 4]
```diff
@@ -1,9 +1,53 @@
...
+    super(props);
+    this.state = {
+      counter: 0,
+    };
+    this.addCounter = this.addCounter.bind(this);
+    this.subtractCounter = this.subtractCounter.bind(this);
+  }
+
+  addCounter() {
+    this.setState((preveState) => ({
```
Comment (rickboot) [inline client/src/App.jsx:16]: is this an abbreviation for previous state? If so, the second e seems odd. Recommended prevState

File: .eslintignore (hunk 2 above)
Comment (ryanminjaeshin) [reply .eslintignore:3]: @rickboot I actually have to ask Eliza about this. I will check it out and comment it here. Thanks!

File: client/src/index.jsx [hunk 5]
```diff
@@ -1,6 +1,6 @@
 /* eslint-disable import/extensions */
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App.jsx';
+import App from './App';
```
Comment (rickboot) [inline client/src/index.jsx:4]: That webconfig section saved us some typing. :)

File: .vscode/settings.json (hunk 3 above)
Comment (ryanminjaeshin) [reply .vscode/settings.json:3]: This is a spell check extension.

File: client/src/App.jsx (hunk 4 above)
Comment (ryanminjaeshin) [reply client/src/App.jsx:16]: @rickboot Make sense. I will edit it. Thanks!

File: client/src/index.jsx (hunk 5 above)
Comment (ryanminjaeshin) [reply client/src/index.jsx:4]: @rickboot 🤩

File: client/test/App.spec.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import App, { Counter } from '../src/App';
+
+describe('App', () => {
+  test('snapshot renders', () => {
+    const component = renderer.create(<App />);
```
Comment (rickboot) [inline client/test/App.spec.js:8]: Cool. A snapshot test.  Is this working, I'd love to see it working.

File: package.json [hunk 7]
```diff
@@ -6,8 +6,10 @@
   "scripts": {
     "react-dev": "webpack -d --watch",
     "server": "nodemon server/index.js",
-    "start" : "webpack-dev-server --open",
-    "seeding": "node -e 'require(\"./database/index.js\").seedingData()'"
+    "start": "webpack-dev-server --open",
```
Comment (rickboot) [inline package.json:9]: We're starting to understand the entire package.json file. :)

File: webpack.config.js [hunk 8]
```diff
@@ -22,5 +22,7 @@ module.exports = {
   },
   devServer: {
     contentBase: path.join(__dirname, '/client/dist'),
```
Comment (rickboot) [inline webpack.config.js:24]: Thanks for this line. Made mine work too.

**Review by rickboot — APPROVED** [review]
Looks good. 

Is that enough comments? Too many. :)

File: .babelrc (hunk 1 above)
Comment (ryanminjaeshin) [reply .babelrc:8]: @rickboot I've commented this on slack.

File: client/test/App.spec.js (hunk 6 above)
Comment (ryanminjaeshin) [reply client/test/App.spec.js:8]: @rickboot Yes. You can check this out in your __snapshots__ folder when you test this. Pretty cool!

**Pull request merged**